.travis.yml

```diff
@@ -36,7 +36,7 @@ notifications:
 
 # Remove /dist/ from .gitignore
 before_deploy:
-  - sed -i '/\/dist\//d' ./.gitignore
```

Comment:
why is this even necessary?
see https://github.com/coala/gh-board/issues/49#issuecomment-396011488